### ISSUE TITLE
Item click inside the InfoWindow

### ISSUE THREAD

**Issue by jasbirsmindfiresolutions** [fullstackreact/google-maps-react#517]
Hi, 

I have added a button in the info window, but i cant perform any action on click of that button.

**Comment by josepBasetis** [comment]
i'm having this same issue. any solution ?

**Comment by itsmejasbirsingh** [comment]
No, even font family gets changed when try to open the info window.

you may look out to an another library, but i did'nt tried it.

https://www.npmjs.com/package/react-google-maps

**Comment by RajThapa7** [comment]
We can work around this by calling a function on the onOpen event on the InfoWindow component; inside that function, we can render the components where we want the click events.
Here is a source to the code sandbox:
https://codesandbox.io/p/sandbox/k0xrxok983
